controller/QtiCreator.php

```
@@ -137,6 +137,12 @@ public function index()
         $config = $this->getCreatorConfig($item);
 
         $this->setData('config', $config->toArray());
+        $this->setData(
+            'itemCommentsEnabled',
+            $this->getFeatureFlagChecker()->isEnabled(
+                FeatureFlagCheckerInterface::FEATURE_FLAG_RESOURCE_COMMENTS_ENABLED
```

PHPStan (2.2.8)

[error] Access to undefined constant oat\tao\model\featureFlag\FeatureFlagCheckerInterface::FEATURE_FLAG_RESOURCE_COMMENTS_ENABLED.


(classConstant.notFound)